### ISSUE TITLE
Documentation links lead to different pages.

### ISSUE THREAD

**Issue by tarasBilobran** [magmax/python-inquirer#338]
In the README there is a button "Read the Docs" that leads to the "https://python-inquirer.readthedocs.io/en/latest/".

![image](https://user-images.githubusercontent.com/71931715/214260888-fdc100e9-6636-4e15-86f9-f10289caba8c.png)

However, if you scroll down a bit, you can see the message "Documentation has been moved to [magmax.org/python-inquirer](https://magmax.org/python-inquirer/).".

In my opinion, it's a bit confusing, could you please take a look at this? If possible, remove the unused documentation link and keep the most recent one.

Thanks!

**Comment by Cube707** [comment]
known issue, see #231. We are still waiting on feedback from @magmax as he controls the webpage.

**Comment by tarasBilobran** [comment]
Sorry, I didn't notice the previous issue.
Thanks, for the update.

**Comment by Cube707** [comment]
duplicate of #231
